garrysmod/gamemodes/terrortown/gamemode/lang/german.lua

```diff
@@ -75,7 +75,7 @@ L.equip_help_cost = "Jedes Ausrüstungsteil, das du kaufst, kostet 1 Credit."
 L.equip_help_carry = "Du kannst nur das kaufen, für das du auch Platz hast."
 L.equip_carry      = "Du kannst diese Ausrüstung tragen."
 L.equip_carry_own  = "Du trägst dieses Teil bereits."
-L.equip_carry_slot = "Du trägst bereits eine Waffe in Slot {slot}."
```

Comment:
Don't change this. "Position" is a bad translation as it has too much meanings.
"Slot" should also be correct. See http://www.duden.de/rechtschreibung/Slot#Bedeutung1 .